.github/workflows/label.yml

```diff
@@ -9,7 +9,7 @@ name: Labeler
 on: [pull_request]
 
 jobs:
-  label:
+  label: label2
```

Comment:
Neglect those changes. Please revert them

Reply:
Do that change alone, that's enough. I didn't say you to make this change. I'm still trying to fix them